packages/racer/src/profiling/config.ts

```diff
@@ -14,6 +23,51 @@ export interface RaceProfileCommand {
   blockedUrlPatterns?: string[];
 }
 
+/**
+ * Defines the configuration and properties we expect for a given Lighthouse run
+ */
+export interface RaceFlowCommand {
+  testFile: string;
+  deviceType?: 'desktop' | 'mobile';
+  chromeFlags?: string[];
+  // extraHeaders?: Record<string, string>;
+  // disableStorageReset?: boolean;
+  // blockedUrlPatterns?: string[];
+}
+
+export interface TestCaseType {
+  connect: Function;
+}
+
+export class FlowContext {
+  jobId: number;
+  testCase: TestCaseType;
+  deviceType = 'desktop';
+  chromeFlags: string[] = [];
+  // disableStorageReset = false;
+  // extraHeaders: Record<string, string> = {};
+  // blockedUrlPatterns: string[] = [];
+
+  constructor(
+    requestedJobId: number,
+    command: RaceFlowCommand,
+    testCase: TestCaseType
+  ) {
+    this.jobId = requestedJobId;
+    this.testCase = testCase;
+    this.deviceType = command.deviceType || this.deviceType;
+    // this.disableStorageReset =
```

Comment:
do we still need this commented out code?

Reply:
I think we'll eventually want to wire these in